### DISCUSSION TITLE
Version 3.4.1 Column sort  doesn't seem to work for views

### DISCUSSION THREAD

**Discussion by rec1968** [pawelsalawa/sqlitestudio#4621] [Q&A]
Windows 10 [Version 10.0.19044.2364]
SqliteStudio 3.4.1

It may just be an artifact of my ignorance but I can't seem to get "Sort by columns" to work with views. I am able to get it to work with tables.

BTW Thanks for SqliteStudion it is an impressive piece of software.

**Comment by pawelsalawa** [comment]
Duplicate of #4607

It's actually already fixed (so today night the weekly build will have it fixed).

**Comment by rec1968** [comment]
Thank you! Sorry that my search missed #4607. Will try harder next time. ;o
